tests/suite.conf

```diff
@@ -28,6 +28,7 @@ kerInvalidTypeValue: invalid type in tiff structure
 kerNotAJpeg : This does not look like a JPEG image
 kerNoImageInInputData: Input data does not contain a valid image
 addition_overflow_message: Overflow in addition
+exiv2exe: exiv2${ENV:binary_extension}
```

Comment:
This is not really worth putting into the global variables, unless it is used in more places.

```suggestion
```

Reply:
I can't figure out how to make it work any other way. I have tried putting `exiv2${ENV:binary_extension}` or `exiv2$binary_extension` in the test file, but it won't expand the variable when the `$` is in the middle of the word. Do you know how to do it?

Reply:
Yeah, `$ENV` doesn't get expanded in tests, only in the config file. However, now you no longer need this variable, so we can remove it altogether?

Reply:
Right now, I am still using it in the `stderr` check. If I add a `compare_stderr` too, then I could remove the global variable.